vendor/libtuntap-master/tuntap-windows.c

```diff
@@ -423,15 +423,11 @@ tuntap_sys_set_ipv4(struct device *dev, t_tun_in_addr *s, uint32_t mask)
    * inline */
   dns.dhcp_opt = 6;
   dns.length   = 4;
-  if (dev->bindaddr)
+  /*if (dev->bindaddr)
     dns.value[0] = dev->bindaddr; /* apparently this doesn't show in network properties,
-                            but it works */
-  else
-    dns.value[0] = htonl(0x7f000001);
-  dns.value[1] = 0;
-
-  ret = DeviceIoControl(dev->tun_fd, TAP_IOCTL_CONFIG_DHCP_SET_OPT, &dns,
-                        sizeof(dns), &dns, sizeof(dns), &len, NULL);
+                            but it works 
+  else*/
```

Comment:
this got commented out due to the refactor, we're back to defaulting to 127.0.0.1 _again_

Reply:
i dont understand why this change is in?

Reply:
kee wasn't able to connect to hidden services last night, going back to our old default fixed it for now

Reply:
this isn't the right fix for that.

Reply:
we can fix it in another PR

Reply:
there i fixed it